### PR TITLE
Add editorconfig file and set rules

### LINKED ISSUE #15: Add EditorConfig
## References

- EditorConfig
https://editorconfig.org/

- Visual Studio Code - EditorConfig extension
https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

## Checklist

- [x] Add EditorConfig extension for VS Code

_Coming soon..._

### PR DESCRIPTION
**Pull request by gmoraleslondono**
Closes #15

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+charset = utf-8
```
Comment (techeverri) [inline .editorconfig]: `charset` should be under the `[*]` (all files) section

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
```
Comment (techeverri) [inline .editorconfig]: Please remove example comment

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+charset = utf-8
+
+# 2 space indentation
+indent_style = space
+indent_size = 2
```
Comment (techeverri) [inline .editorconfig]: Move under the `[*]` (all files) section

**Review by techeverri — CHANGES_REQUESTED** [review]
I'd suggest:
- removing all the comments and
- sorting alphabetically the properties under the `[*]` (all files) section

**Comment by gmoraleslondono** [comment]
Thank you! I did the changes

**Review by techeverri — COMMENTED** [review]
Does any of the properties conflict with Prettier? :thinking:

**Review by techeverri — APPROVED** [review]
Good job! 👍

**Comment by techeverri** [comment]
In case there is any conflict between Prettier and the EditorConfig we can fix it later in a separate pull request

**Comment by gmoraleslondono** [comment]
I checked and it doesn't have conflicts with prettier.

**Pull request merged**